config/config.go

```diff
@@ -1,4 +1,5 @@
 // Package config implements functions to assist with attribute evaluation in the SLAM service
+// Warning: The code in this package might be transient. Stability is not guaranteed.
```

Comment:
I wonder if the warning in the README would be sufficient... it will be one less thing to remember to do in any new files, and I'm not sure adding the warning to every file will provide more value.

Reply:
That's true - though I wonder if we should leave it in for two reasons:
1. When we added "This is an Experimental package" to the original slam repo, we agreed to add it both to the readme and to the package description. 
2. Adding it in the package description will make it visible in go.viam.com in the package overview, for example like here: https://pkg.go.dev/go.viam.com/rdk@v0.2.33/services/slam/builtin
    * I think this is helpful for users who might search for help on certain functions and will be directed towards the documentation. They can then derive from the Overview that this package is not stable.

Given those reasons, do you still think that the README might be sufficient?